app/src/pages/react-uswds-hello.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Alert,
+  Button,
+  Grid,
+  GridContainer,
+  Icon,
+} from "@trussworks/react-uswds";
+import Link from "next/link";
+import React from "react";
+
+/** Example page showing use of react-uswds components. Can be deleted once you're on a roll. */
+const HelloWorld = () => {
+  return (
+    <GridContainer>
+      <Grid row gap={5}>
```

Comment:
❓ I wanted to add a prop to this row so that the classname 'grid-row-auto' would populate (so the alert would take up the whole screen). Couldn't figure out from the docs.

Reply:
Each row should have at least one "column" (`col`), which should fix this. Since there's only one column, you could get rid of the `gap` prop, since that controls spacing between columns.


```tsx
<Grid row>
  <Grid col="fill">
      <Alert ...
```